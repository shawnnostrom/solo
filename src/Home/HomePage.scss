.home-title{
  color: #30C3FF;
}
.list-title {
  display: flex;
  justify-content: center;
  .list-position-type {
    color: #30C3FF;
    margin-left:5px;
  }
}
.component-wrapper {
  display:flex;
  justify-content: space-evenly;
  .list-wrapper {
    width:100%;
  }
}
.details-component-wrapper {
  min-width: 50%;
}