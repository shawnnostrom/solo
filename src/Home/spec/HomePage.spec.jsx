import React from 'react';
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import HomePage from '../index'

it('should match snapshot', () => {
  const { asFragment } = render(
    <HomePage />
  )
  expect(asFragment()).toMatchSnapshot()
})