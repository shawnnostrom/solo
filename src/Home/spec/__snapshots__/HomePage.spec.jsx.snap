// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="home-title"
    >
      Who's My Representative?
    </h1>
    <div>
      <select
        name="Position"
      >
        <option>
          Select Type
        </option>
        <option
          value="Representatives"
        >
          Representative
        </option>
        <option
          value="senators"
        >
          Senator
        </option>
      </select>
      <select
        name="States"
      >
        <option>
          Select State
        </option>
        <option
          value="AL"
        >
          Alabama
        </option>
        <option
          value="AK"
        >
          Alaska
        </option>
        <option
          value="AS"
        >
          American Samoa
        </option>
        <option
          value="AZ"
        >
          Arizona
        </option>
        <option
          value="AR"
        >
          Arkansas
        </option>
        <option
          value="CA"
        >
          California
        </option>
        <option
          value="CO"
        >
          Colorado
        </option>
        <option
          value="CT"
        >
          Connecticut
        </option>
        <option
          value="DE"
        >
          Delaware
        </option>
        <option
          value="DC"
        >
          District Of Columbia
        </option>
        <option
          value="FM"
        >
          Federated States Of Micronesia
        </option>
        <option
          value="FL"
        >
          Florida
        </option>
        <option
          value="GA"
        >
          Georgia
        </option>
        <option
          value="GU"
        >
          Guam
        </option>
        <option
          value="HI"
        >
          Hawaii
        </option>
        <option
          value="ID"
        >
          Idaho
        </option>
        <option
          value="IL"
        >
          Illinois
        </option>
        <option
          value="IN"
        >
          Indiana
        </option>
        <option
          value="IA"
        >
          Iowa
        </option>
        <option
          value="KS"
        >
          Kansas
        </option>
        <option
          value="KY"
        >
          Kentucky
        </option>
        <option
          value="LA"
        >
          Louisiana
        </option>
        <option
          value="ME"
        >
          Maine
        </option>
        <option
          value="MH"
        >
          Marshall Islands
        </option>
        <option
          value="MD"
        >
          Maryland
        </option>
        <option
          value="MA"
        >
          Massachusetts
        </option>
        <option
          value="MI"
        >
          Michigan
        </option>
        <option
          value="MN"
        >
          Minnesota
        </option>
        <option
          value="MS"
        >
          Mississippi
        </option>
        <option
          value="MO"
        >
          Missouri
        </option>
        <option
          value="MT"
        >
          Montana
        </option>
        <option
          value="NE"
        >
          Nebraska
        </option>
        <option
          value="NV"
        >
          Nevada
        </option>
        <option
          value="NH"
        >
          New Hampshire
        </option>
        <option
          value="NJ"
        >
          New Jersey
        </option>
        <option
          value="NM"
        >
          New Mexico
        </option>
        <option
          value="NY"
        >
          New York
        </option>
        <option
          value="NC"
        >
          North Carolina
        </option>
        <option
          value="ND"
        >
          North Dakota
        </option>
        <option
          value="MP"
        >
          Northern Mariana Islands
        </option>
        <option
          value="OH"
        >
          Ohio
        </option>
        <option
          value="OK"
        >
          Oklahoma
        </option>
        <option
          value="OR"
        >
          Oregon
        </option>
        <option
          value="PW"
        >
          Palau
        </option>
        <option
          value="PA"
        >
          Pennsylvania
        </option>
        <option
          value="PR"
        >
          Puerto Rico
        </option>
        <option
          value="RI"
        >
          Rhode Island
        </option>
        <option
          value="SC"
        >
          South Carolina
        </option>
        <option
          value="SD"
        >
          South Dakota
        </option>
        <option
          value="TN"
        >
          Tennessee
        </option>
        <option
          value="TX"
        >
          Texas
        </option>
        <option
          value="UT"
        >
          Utah
        </option>
        <option
          value="VT"
        >
          Vermont
        </option>
        <option
          value="VI"
        >
          Virgin Islands
        </option>
        <option
          value="VA"
        >
          Virginia
        </option>
        <option
          value="WA"
        >
          Washington
        </option>
        <option
          value="WV"
        >
          West Virginia
        </option>
        <option
          value="WI"
        >
          Wisconsin
        </option>
        <option
          value="WY"
        >
          Wyoming
        </option>
      </select>
      <button>
        Submit
      </button>
    </div>
    <div
      class="component-wrapper"
    />
  </div>
</DocumentFragment>
`;

exports[`Layout should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="home-title"
    >
      Who's My Representative?
    </h1>
    <div>
      <select
        name="Position"
      >
        <option>
          Select Type
        </option>
        <option
          value="Representatives"
        >
          Representative
        </option>
        <option
          value="senators"
        >
          Senator
        </option>
      </select>
      <select
        name="States"
      >
        <option>
          Select State
        </option>
        <option
          value="AL"
        >
          Alabama
        </option>
        <option
          value="AK"
        >
          Alaska
        </option>
        <option
          value="AS"
        >
          American Samoa
        </option>
        <option
          value="AZ"
        >
          Arizona
        </option>
        <option
          value="AR"
        >
          Arkansas
        </option>
        <option
          value="CA"
        >
          California
        </option>
        <option
          value="CO"
        >
          Colorado
        </option>
        <option
          value="CT"
        >
          Connecticut
        </option>
        <option
          value="DE"
        >
          Delaware
        </option>
        <option
          value="DC"
        >
          District Of Columbia
        </option>
        <option
          value="FM"
        >
          Federated States Of Micronesia
        </option>
        <option
          value="FL"
        >
          Florida
        </option>
        <option
          value="GA"
        >
          Georgia
        </option>
        <option
          value="GU"
        >
          Guam
        </option>
        <option
          value="HI"
        >
          Hawaii
        </option>
        <option
          value="ID"
        >
          Idaho
        </option>
        <option
          value="IL"
        >
          Illinois
        </option>
        <option
          value="IN"
        >
          Indiana
        </option>
        <option
          value="IA"
        >
          Iowa
        </option>
        <option
          value="KS"
        >
          Kansas
        </option>
        <option
          value="KY"
        >
          Kentucky
        </option>
        <option
          value="LA"
        >
          Louisiana
        </option>
        <option
          value="ME"
        >
          Maine
        </option>
        <option
          value="MH"
        >
          Marshall Islands
        </option>
        <option
          value="MD"
        >
          Maryland
        </option>
        <option
          value="MA"
        >
          Massachusetts
        </option>
        <option
          value="MI"
        >
          Michigan
        </option>
        <option
          value="MN"
        >
          Minnesota
        </option>
        <option
          value="MS"
        >
          Mississippi
        </option>
        <option
          value="MO"
        >
          Missouri
        </option>
        <option
          value="MT"
        >
          Montana
        </option>
        <option
          value="NE"
        >
          Nebraska
        </option>
        <option
          value="NV"
        >
          Nevada
        </option>
        <option
          value="NH"
        >
          New Hampshire
        </option>
        <option
          value="NJ"
        >
          New Jersey
        </option>
        <option
          value="NM"
        >
          New Mexico
        </option>
        <option
          value="NY"
        >
          New York
        </option>
        <option
          value="NC"
        >
          North Carolina
        </option>
        <option
          value="ND"
        >
          North Dakota
        </option>
        <option
          value="MP"
        >
          Northern Mariana Islands
        </option>
        <option
          value="OH"
        >
          Ohio
        </option>
        <option
          value="OK"
        >
          Oklahoma
        </option>
        <option
          value="OR"
        >
          Oregon
        </option>
        <option
          value="PW"
        >
          Palau
        </option>
        <option
          value="PA"
        >
          Pennsylvania
        </option>
        <option
          value="PR"
        >
          Puerto Rico
        </option>
        <option
          value="RI"
        >
          Rhode Island
        </option>
        <option
          value="SC"
        >
          South Carolina
        </option>
        <option
          value="SD"
        >
          South Dakota
        </option>
        <option
          value="TN"
        >
          Tennessee
        </option>
        <option
          value="TX"
        >
          Texas
        </option>
        <option
          value="UT"
        >
          Utah
        </option>
        <option
          value="VT"
        >
          Vermont
        </option>
        <option
          value="VI"
        >
          Virgin Islands
        </option>
        <option
          value="VA"
        >
          Virginia
        </option>
        <option
          value="WA"
        >
          Washington
        </option>
        <option
          value="WV"
        >
          West Virginia
        </option>
        <option
          value="WI"
        >
          Wisconsin
        </option>
        <option
          value="WY"
        >
          Wyoming
        </option>
      </select>
      <button>
        Submit
      </button>
    </div>
    <div
      class="component-wrapper"
    />
  </div>
</DocumentFragment>
`;

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="home-title"
    >
      Who's My Representative?
    </h1>
    <div>
      <select
        name="Position"
      >
        <option>
          Select Type
        </option>
        <option
          value="Representatives"
        >
          Representative
        </option>
        <option
          value="senators"
        >
          Senator
        </option>
      </select>
      <select
        name="States"
      >
        <option>
          Select State
        </option>
        <option
          value="AL"
        >
          Alabama
        </option>
        <option
          value="AK"
        >
          Alaska
        </option>
        <option
          value="AS"
        >
          American Samoa
        </option>
        <option
          value="AZ"
        >
          Arizona
        </option>
        <option
          value="AR"
        >
          Arkansas
        </option>
        <option
          value="CA"
        >
          California
        </option>
        <option
          value="CO"
        >
          Colorado
        </option>
        <option
          value="CT"
        >
          Connecticut
        </option>
        <option
          value="DE"
        >
          Delaware
        </option>
        <option
          value="DC"
        >
          District Of Columbia
        </option>
        <option
          value="FM"
        >
          Federated States Of Micronesia
        </option>
        <option
          value="FL"
        >
          Florida
        </option>
        <option
          value="GA"
        >
          Georgia
        </option>
        <option
          value="GU"
        >
          Guam
        </option>
        <option
          value="HI"
        >
          Hawaii
        </option>
        <option
          value="ID"
        >
          Idaho
        </option>
        <option
          value="IL"
        >
          Illinois
        </option>
        <option
          value="IN"
        >
          Indiana
        </option>
        <option
          value="IA"
        >
          Iowa
        </option>
        <option
          value="KS"
        >
          Kansas
        </option>
        <option
          value="KY"
        >
          Kentucky
        </option>
        <option
          value="LA"
        >
          Louisiana
        </option>
        <option
          value="ME"
        >
          Maine
        </option>
        <option
          value="MH"
        >
          Marshall Islands
        </option>
        <option
          value="MD"
        >
          Maryland
        </option>
        <option
          value="MA"
        >
          Massachusetts
        </option>
        <option
          value="MI"
        >
          Michigan
        </option>
        <option
          value="MN"
        >
          Minnesota
        </option>
        <option
          value="MS"
        >
          Mississippi
        </option>
        <option
          value="MO"
        >
          Missouri
        </option>
        <option
          value="MT"
        >
          Montana
        </option>
        <option
          value="NE"
        >
          Nebraska
        </option>
        <option
          value="NV"
        >
          Nevada
        </option>
        <option
          value="NH"
        >
          New Hampshire
        </option>
        <option
          value="NJ"
        >
          New Jersey
        </option>
        <option
          value="NM"
        >
          New Mexico
        </option>
        <option
          value="NY"
        >
          New York
        </option>
        <option
          value="NC"
        >
          North Carolina
        </option>
        <option
          value="ND"
        >
          North Dakota
        </option>
        <option
          value="MP"
        >
          Northern Mariana Islands
        </option>
        <option
          value="OH"
        >
          Ohio
        </option>
        <option
          value="OK"
        >
          Oklahoma
        </option>
        <option
          value="OR"
        >
          Oregon
        </option>
        <option
          value="PW"
        >
          Palau
        </option>
        <option
          value="PA"
        >
          Pennsylvania
        </option>
        <option
          value="PR"
        >
          Puerto Rico
        </option>
        <option
          value="RI"
        >
          Rhode Island
        </option>
        <option
          value="SC"
        >
          South Carolina
        </option>
        <option
          value="SD"
        >
          South Dakota
        </option>
        <option
          value="TN"
        >
          Tennessee
        </option>
        <option
          value="TX"
        >
          Texas
        </option>
        <option
          value="UT"
        >
          Utah
        </option>
        <option
          value="VT"
        >
          Vermont
        </option>
        <option
          value="VI"
        >
          Virgin Islands
        </option>
        <option
          value="VA"
        >
          Virginia
        </option>
        <option
          value="WA"
        >
          Washington
        </option>
        <option
          value="WV"
        >
          West Virginia
        </option>
        <option
          value="WI"
        >
          Wisconsin
        </option>
        <option
          value="WY"
        >
          Wyoming
        </option>
      </select>
      <button>
        Submit
      </button>
    </div>
    <div
      class="component-wrapper"
    />
  </div>
</DocumentFragment>
`;
